README.md

```diff
@@ -1,60 +1,73 @@
+The documentation is available at [doc.akka.io](http://doc.akka.io),
+for [Scala](http://doc.akka.io/docs/akka/current/scala/http/) and
+[Java](http://doc.akka.io/docs/akka/current/java/http/).
 
 
 Community
 ---------
 You can join these groups and chats to discuss and ask Akka related questions:
 
-- Mailing list: [![google groups: akka-user](https://img.shields.io/badge/group%3A-akka--user-blue.svg?style=flat-square)](https://groups.google.com/forum/#!forum/akka-user)
-- Chat room about *using* Akka: [![gitter: akka/akka](https://img.shields.io/badge/gitter%3A-akka%2Fakka-blue.svg?style=flat-square)](https://gitter.im/akka/akka)
-- Issue tracker: [![github: akka/akka](https://img.shields.io/badge/github%3A-issues-blue.svg?style=flat-square)](https://github.com/akka/akka/issues)
+- Mailing list: [![google groups: akka-user][groups-user-badge]][groups-user]
+- Chat room about *using* Akka HTTP: [![gitter: akka/akka][gitter-user-badge]][gitter-user]
+- Q&A: [![stackoverflow: #akka-http][stackoverflow-badge]][stackoverflow]
+- Issue tracker: [![github: akka/akka-http][github-issues-badge]][github-issues]
 
 In addition to that, you may enjoy following:
 
 - The [news](http://akka.io/news) section of the page, which is updated whenever a new version is released
 - The [Akka Team Blog](http://blog.akka.io)
 - [@akkateam](https://twitter.com/akkateam) on Twitter
-- Questions tagged [#akka on StackOverflow](http://stackoverflow.com/questions/tagged/akka)
-- Projects built with Akka: [![akka-dependency-badge]][akka-dependency-scaladex]
+- Projects built with Akka HTTP: [![Built with Akka HTTP][scaladex-badge]][scaladex-projects]
+
+[groups-user-badge]:   https://img.shields.io/badge/group%3A-akka--user-blue.svg?style=flat-square
+[groups-user]:         https://groups.google.com/forum/#!forum/akka-user
+[gitter-user-badge]:   https://img.shields.io/badge/gitter%3A-akka%2Fakka-blue.svg?style=flat-square
+[gitter-user]:         https://gitter.im/akka/akka
+[stackoverflow-badge]: https://img.shields.io/badge/stackoverflow%3A-akka--http-blue.svg?style=flat-square
+[stackoverflow]:       http://stackoverflow.com/questions/tagged/akka-http
+[github-issues-badge]: https://img.shields.io/badge/github%3A-issues-blue.svg?style=flat-square
+[github-issues]:       https://github.com/akka/akka-http/issues
+[scaladex-badge]:      https://index.scala-lang.org/count.svg?q=dependencies:akka/akka-http*&subject=scaladex:&color=blue&style=flat-square
+[scaladex-projects]:   https://index.scala-lang.org/search?q=dependencies:akka/akka-http*
 
 Contributing
 ------------
 Contributions are *very* welcome!
 
-If you see an issue that you'd like to see fixed, the best way to make it happen is to help out by submitting a PullRequest implementing it.
+If you see an issue that you'd like to see fixed, the best way to make it happen is to help out by submitting a pull request.
```

Comment:
Here could be a link saying:

For ideas of where to contributing [these tickets](https://github.com/akka/akka-http/issues?q=is%3Aissue+is%3Aopen+label%3Acommunity) are a good starting point.